telegram/src/main/kotlin/com/github/kotlintelegrambot/entities/InlineKeyboardMarkup.kt

```diff
@@ -14,12 +14,22 @@ data class InlineKeyboardMarkup internal constructor(
 ) : ReplyMarkup {
 
     init {
-        // Buttons of type [Pay] must always be the first button in the first row.
+        validatePriorityButtonsForType<InlineKeyboardButton.Pay>()
+        validatePriorityButtonsForType<InlineKeyboardButton.CallbackGameButtonType>()
+    }
+
+    // Priority buttons must always be the first button in the first row.
+    private inline fun <reified T> validatePriorityButtonsForType() {
         val flattenedButtons = inlineKeyboard.flatten()
-        val payButtons = flattenedButtons.filterIsInstance<InlineKeyboardButton.Pay>()
-        require(payButtons.size <= 1) { "Can't have more than one pay button per inline keyboard" }
-        require(payButtons.size != 1 || flattenedButtons.firstOrNull() is InlineKeyboardButton.Pay) {
-            "Pay buttons must always be the first button in the first row"
+        val filteredButtons = flattenedButtons.filterIsInstance<T>()
+        val typeName = T::class.simpleName
```

Comment:
Not really important: Should we access the class name only when it's needed and not beforehand? I guess most of the cases will be valid inline keyboards that don't need to load the button type name.

I mean, I don't think it's going to make a difference. I'm happy to merge this as is (I'll leave the comment for some minutes and I'll merge it if you don't see the comment in that time).